### ISSUE TITLE
Cannot read property 'kind' of undefined

### ISSUE THREAD

**Issue by seangwright** [daidodo/tsimportsorter#8]
With the latest version of the extension I'm seeing this error in VS Code:

```
Error found: Cannot read property 'kind' of undefined. 
If you believe this is a bug, please report on https://github.com/daidodo/tsimportsorter
```

I'm using `"typescript": "3.8.3",`.

My project dependencies have not changed since everything was previously working, only the extension version.

I updated my `import-sorter.json` to use the renamed rules:

```json
"maxLineLength": 120,
"maxBindingNamesPerLine": 8,
"maxDefaultAndBindingNamesPerLine": 8,
"maxNamesPerWrappedLine": 4,
"autoFormat": "onSave",
```

But that hasn't resolved anything.

**Comment by Kizmar** [comment]
Same issue here. Same version of TypeScript. This appears to have started right after the extension auto-updated to 2.0.2.

**Comment by robertowrethink** [comment]
Same issue here. I use `react": "16.8.3` and `react-native": "0.59.10`.

**Comment by daidodo** [comment]
Can someone provide a code example for the issue? Thanks!

**Comment by Kizmar** [comment]
@daidodo I get the same error from a brand new create-react-app instance.

**Comment by seangwright** [comment]
Here's one you can try - brand new, untouched Angular 9 application.

https://github.com/seangwright/ng914

`git clone`
`cd ng914`
`npm i`
`code .`

Now save the `src/app/app.module.ts` and see the error in the VS Code toast in the bottom right.

**Comment by daidodo** [comment]
@Kizmar , it's working fine in my end.
<img width="600" alt="image" src="https://user-images.githubusercontent.com/8170176/81101166-a25f8980-8f05-11ea-99f9-7a4a3ea055ce.png">

**Comment by seangwright** [comment]
I just installed VS Code Insiders (v1.45.0) (My initial error occurred in normal VS Code v1.44.2) and have 0 extensions installed other than `dozerg.tsimportsorter`.

When I save the `src/app/app.module.ts` I see the error that everyone else sees.

(Btw, just want to mention I ♥ this extension and your work. It's just that since I've made this extension part of my workflow for lots of apps, this issue is a little bit of a show stopper)

**Comment by Kizmar** [comment]
Hmm... so I reverted step by step back to version 1.2.12 and still get the error. Tried removing all of the custom values from `settings.json`. Tried uninstalling it completely and reinstalling. Still get the error in every case.

**Comment by daidodo** [comment]
Hi @seangwright and @Kizmar, thanks for the info!

I'm in Mac. I [uninstalled VSCode completely](https://stackoverflow.com/questions/42603103/how-to-completely-uninstall-vscode-on-mac), re-installed VSCode (stable version) and only this plugin (2.0.2), but still couldn't reproduce the error

Now I'm going to try in Window 10.

**Comment by Kizmar** [comment]
@daidodo Sweet. I just ran upstairs and tested vscode on my Mac and verified the error did not show up there. :)

Thank you for your work on this extension. I love it!

**Comment by daidodo** [comment]
Got closed by github automatically after I pushed the code. The issue should be fixed in **v2.0.3**.

### Root Cause
The issue came from an exception thrown by `program.getSemanticDiagnostics(sourceFile)` ([code](https://github.com/daidodo/tsimportsorter/commit/46c18fd4ef29f8af20ac791289865dded31c2730?diff=split#diff-57152deda617a294f6147233aaca6715L34)) on Windows.

### Fix
In short term, I reinstated manual parsing for identifiers deprecated in v1.2.9. In long term, I need to find out what causes the exception.

Thank you all for the helps in this bug hunting!

**Comment by seangwright** [comment]
I just installed the latest version, saved a file and all the formatting worked correctly.

Thanks again! Love this extension (especially because we often use mutli-root workspaces).

🙏
